include/Game/Cave/RandMapMgr.h

```diff
@@ -23,13 +23,25 @@ struct EnemyNode;
 struct GateNode;
 struct ItemNode;
 struct MapNode;
+struct MapUnitGenerator;
+struct BaseGen;
 struct RandEnemyUnit;
-struct RandPlantUnit;
+struct RandPlantUnit {
```

Comment:
This header needs to be cleaned up, but we can do that after the merge is complete, given it's matching